web/src/pages/orders/index.page.jsx

```diff
@@ -27,6 +28,11 @@ const Order = () => {
           <Loading />
         : <Table headers={headers} data={formattedOrders} showActions={true} />}
       </section>
+
+      <div className='m-8'>
+      <Paginations />
+      </div>
```

Comment:
Proper indention and spacing to make it readable
```suggestion
      <div className='m-8'>
          <Paginations />
      </div>
```